src/scantailor/app/ui/MainWindow.py

```
@@ -0,0 +1,328 @@
+
+
+class MainWindowManager:
+    """Controller Class for the Main Application window for ScanTailor.
+
+    Note: This class is not a QMainWindow.
+    When using QtUiLoader to load a QMainWindow based .ui file, and setting the parent
+    as a QMainWindow, we get weird behavior.
+    e.g. https://stackoverflow.com/questions/53828666/pyside2-qmainwindow-loaded-from-ui-file-not-triggering-window-events
+    """
+
+    def __init__(self):
```

Pylint (3.3.7)

[refactor] Too many statements (90/50)

(R0915)